portal-impl/src/com/liferay/portal/service/impl/OrganizationLocalServiceImpl.java

```diff
@@ -150,18 +150,7 @@ public boolean addGroupOrganization(long groupId, long organizationId) {
 	public boolean addGroupOrganization(
 		long groupId, Organization organization) {
 
-		if (!super.addGroupOrganization(groupId, organization)) {
-			return false;
-		}
-
-		try {
-			reindexUsers(organization);
-		}
-		catch (PortalException portalException) {
-			throw new SystemException(portalException);
-		}
-
-		return true;
+		return addGroupOrganization(groupId, organization.getOrganizationId());
 	}
 
```

Comment:
@marianoalvarosaiz, please don't do the change in OrganizationLocalServiceImpl, UserGroupLocalServiceImpl and UserLocalServiceImpl classes. Because when using ID to reindex, it will fetch model again. For example, please see the [logic](https://github.com/liferay-core-infra/liferay-portal/blob/f9026779e26e34ed5cd95559045b558a7e8f572b/portal-impl/src/com/liferay/portal/service/impl/OrganizationLocalServiceImpl.java#L2493)